src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/spec/SpecGerritTriggerHudsonTest.java

```diff
@@ -465,10 +466,13 @@ public void testTripleTriggeredBuildWithProjects() throws Exception {
      *
      * @throws Exception if so.
      */
+    @Test
     @LocalData
     public void testBuildLatestPatchsetOnly() throws Exception {
         GerritServer gerritServer = PluginImpl.getInstance().getServer(PluginImpl.DEFAULT_SERVER_NAME);
         ((Config)gerritServer.getConfig()).setGerritBuildCurrentPatchesOnly(true);
+        ((Config)gerritServer.getConfig()).setGerritAbortManualPatchsets(false);
```

Comment:
there are four different states to test here;
* both ```false```
* both ```true```
* one ```false``` and the other ```true```